lua/alex/lazy/mason-tool-installer.lua

```
@@ -0,0 +1,19 @@
+return {
+  {
+    "WhoIsSethDaniel/mason-tool-installer.nvim",
+    config = function()
+      require("mason-tool-installer").setup {
+        ensure_installed = {
+          { 'gopls', condition = function() return vim.fn.executable('go') == 1  end },
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)